readthedocs/rtd_tests/tests/test_celery.py

```diff
@@ -356,8 +356,8 @@ def test_send_build_status_task_with_remote_repo(self, send_build_status):
         )
         self.assertEqual(Message.objects.filter(user=self.eric).count(), 0)
 
-    @patch('readthedocs.projects.tasks.GitHubService.send_build_status')
-    def test_send_build_status_task_with_social_account(self, send_build_status):
+    @patch('readthedocs.oauth.services.github.GitHubService.send_build_status')
```

Comment:
Why did this change? It seems like it was wrong before?

Reply:
Because we are not directly importing `GitHubService` to the tasks we are using `project.git_service_class()`

Reply:
It was working before because I forgot to remove an unused import of `GitHubService`